src/exporters/cds.py

```diff
@@ -75,38 +76,56 @@ def create_area(region: Region) -> str:
         return "/".join(["{:.3f}".format(x) for x in x])
 
     @staticmethod
-    def make_filename(dataset: str, selection_request: Dict) -> str:
-        """Makes the appropriate filename for a CDS export
-        """
-        date_str = ''
-        if 'year' in selection_request:
-            years = selection_request['year']
-            if len(years) > 1:
-                warnings.warn('More than 1 year of data being exported! '
+    def _filename_from_selection_request(time_array: List, time: str) -> str:
+        if len(time_array) > 1:
+            if time in {'month', 'year'}:
+                warnings.warn(f'More than 1 {time} of data being exported! '
                               'Export times may be significant.')
-                years.sort()
-                date_str = f'{years[0]}_{years[-1]}'
-            else:
-                date_str = str(years[0])
-        elif 'date' in selection_request:
```

Comment:
@gabrieltseng so is `date` the date that the user made the download? or the `dates` of the data